src/components/manageParticipants/molecules/AddParticipantsForm.tsx

```diff
@@ -74,6 +74,16 @@ const AddParticipantsForm: React.FC<IProps> = ({ meetingId, participants, setPar
           },
         });
         setParticipants([...participants, { email: lowerCaseEmail, role: inputRole, isVotingEligible: true }]);
+      } else if (meetingId) {
```

Comment:
I think the code would be more readable if we swapped `meetingId` with `emailAlreadyAdded` here. I think we primarily check `meetingId` on line 63 to ensure it is a valid value for the query.